### ISSUE TITLE
Test "Last Diff Backup Times" does not honor config value "skip.diffbackuptest"

### ISSUE THREAD

**Issue by tootblan** [sqlcollaborative/dbachecks#297]
![image](https://user-images.githubusercontent.com/25352226/36560938-dc5fa1a8-17d7-11e8-822f-bff31d12dd75.png)

There are 18 databases on this server, and each one fails this test with:
```powershell
[-] ReportServer diff backups on SQLADMIN Should Be less than 25 hours 45ms
	Expected 2018-02-21T12:55:03.3587274-06:00 to be greater than the actual value, because Taking regular backups is extraordinarily important, but got 0001-01-01T00:00:00.0000000.
	265: $psitem.LastDiffBackup | Should -BeGreaterThan (Get-Date).AddHours(- ($maxdiff)) -Because 'Taking regular backups is extraordinarily important'
		at <ScriptBlock>, C:\Users\powershell\Documents\WindowsPowerShell\Modules\dbachecks\1.0.0\checks\Database.Tests.ps1: line 265
```
None of the 18 databases on this server has ever had a DIFF backup. We simply don't use them on this server.

**Comment by ClaudioESSilva** [comment]
Thanks for reporting @tootblan.
My guest is, when there is no value the default (for DateTime fields) become as 0001-01-01T00:00:00.0000000 so, we need to check that the value is not this one also to skip the yes for that database.

We will fix this in the next days.

**Comment by SQLDBAWithABeard** [comment]
There is no call to check the configuration item here

https://github.com/michalporeba/dbachecks/blob/master/checks/Database.Tests.ps1#L257

Thank you - this should be an easy fix @ClaudioESSilva  can you do??

**Comment by ClaudioESSilva** [comment]
Done @SQLDBAWithABeard.
@tootblan please test it with PR #349

Thanks

**Comment by tootblan** [comment]
Tested and confirmed as working. Thanks @ClaudioESSilva!

**Comment by SQLDBAWithABeard** [comment]
Thank you both merging now

**Comment by SQLDBAWithABeard** [comment]
Fixed with #349
